crates/shared/src/bad_token/token_owner_finder/seasolver.rs

```diff
@@ -0,0 +1,45 @@
+use super::TokenOwnerProposing;
+use anyhow::Result;
+use ethcontract::H160;
+
+pub struct SeasolverTokenOwnerFinder {
```

Comment:
This implementation is not specific to the `SeaSolver` so naming it more generically would make sense to me.
Maybe simply `TokenOwnerList`?

Reply:
Also agree this can be a general `TokenOwnerList` and doesn't have to be Sea Solver specific.

Reply:
Initially I though having Seasolver specific implementation would better follow OpenClosed principle. But I do agree having general name is better suited at this moment. Fixed.